client/components/Navlist.jsx

```diff
@@ -0,0 +1,31 @@
+/* eslint react/prop-types: 0 */
+
+import React from 'react';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemIcon from '@material-ui/core/ListItemIcon';
+import ListItemText from '@material-ui/core/ListItemText';
+import { Link } from 'react-router-dom';
+import List from '@material-ui/core/List';
+
+const drawerItems = [
+  { route: '/signup', class: 'fas fa-user-plus', text: 'Sign Up' },
```

Comment:
Should this be the same as `Navbar`'s

Reply:
this is now declared in Navbar.jsx and passed to Navlist.jsx so that they both use the same obj array. 
![image](https://user-images.githubusercontent.com/35232697/107263407-4df2b880-6a07-11eb-9d53-fb7ae3172e8e.png)